lib/feedOperations.js

```diff
@@ -0,0 +1,48 @@
+
+module.exports.appendTo = appendableFeed
+
+function appendableFeed (core, feedId) {
+  var refFeed = core.get(feedId)
+  var newFeed = core.add()
+  var blockNumber = 0
+
+  var copyOriginal = function (resolve, reject) {
+    for (var i = 0; i < refFeed.blocks; i++) {
+      refFeed.get(i, function (err, block) {
+        if (err) {
+          console.log(err)
+          reject()
+        }
+
+        newFeed.append(block, function () {
+          blockNumber++
+          console.log('Copying block ' + blockNumber + '/' + newFeed.blocks + ' from original feed')
+
+          if (blockNumber === refFeed.blocks) {
+            resolve()
+          }
+        })
+      })
+    }
```

Comment:
this might append blocks in the wrong order as this is async. you probably wanna do this

``` js
loop(null)
function loop (err) {
  if (err) return reject(err)
  refFeed.get(newFeed.blocks, function (err, block) {
    if (err) return cb(err)
    if (!block) return resolve() // done
    newFeed.append(block, loop)
  })
}
```